cmd/scaffold.go

```diff
@@ -180,6 +183,13 @@ func (fnb *FlowNodeBuilder) BaseFlags() {
 	fnb.flags.UintVar(&fnb.BaseConfig.SyncCoreConfig.MaxRequests, "sync-max-requests", defaultConfig.SyncCoreConfig.MaxRequests, "the maximum number of requests we send during each scanning period")
 
 	fnb.flags.Uint64Var(&fnb.BaseConfig.ComplianceConfig.SkipNewProposalsThreshold, "compliance-skip-proposals-threshold", defaultConfig.ComplianceConfig.SkipNewProposalsThreshold, "threshold at which new proposals are discarded rather than cached, if their height is this much above local finalized height")
+
+	// unicast stream handler rate limits
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastStreamCreationRateLimit, "unicast-stream-rate-limit", 0, "amount of unicast streams that can be created by a single peer per second")
+	fnb.flags.IntVar(&fnb.BaseConfig.UnicastStreamCreationBurstLimit, "unicast-stream-burst-limit", 0, "amount of unicast streams that can be created by a single peer at once")
```

Comment:
I don't think this config is needed, since you can create at most 1 stream in a single invocation anyway

Reply:
It is needed, burst limit is the bucket size in the token bucket rate limiting algorithm, so 1 would indicate a token bucket size of 1 which allow a user to claim a single token. Rate limit is the rate at which this bucket fills up.